libblkid-rs-sys/build.rs

```diff
@@ -1,8 +1,31 @@
 use bindgen::Builder;
 
-use std::{env, path::PathBuf};
+use std::{
+    env,
+    path::{Path, PathBuf},
+};
+
+#[cfg(target_os = "macos")]
+fn fix_include(include: &PathBuf) -> &Path {
+    if include.ends_with("include/blkid") {
+        include.parent().unwrap()
+    } else {
+        include
+    }
+}
```

Comment:
I would almost prefer a modification to the header file that uses C macros to conditionally include either `blkid/blkid.h` or `blkid.h`. To me, that feels less brittle.

Reply:
I was thinking about this before but wasn't sure how to actually do it.

Reply:
I'm going to look into the pkg-config file provided by MacOS. If we can confirm that the file distributed for Mac references `blkid.h`, that is enough for me. Would that be acceptable to you?

Reply:
I'm OK either way. I was just following your logic of not wanting anything hardcoded to a particular build environment. This pkg-config is installed with the Homebrew package `util-linux`, so it might be Homebrew-specific. I haven't tried anything else (like MacPorts).

```
$ cat /opt/homebrew/opt/util-linux/lib/pkgconfig/blkid.pc
prefix=/opt/homebrew/Cellar/util-linux/2.40.4
exec_prefix=/opt/homebrew/Cellar/util-linux/2.40.4
libdir=/opt/homebrew/Cellar/util-linux/2.40.4/lib
includedir=/opt/homebrew/Cellar/util-linux/2.40.4/include

Name: blkid
Description: Block device id library
Version: 2.40.4
Cflags: -I${includedir}/blkid
Libs: -L${libdir} -lblkid
```

Reply:
I just looked at the pkg-config file that seems to be included everywhere and discovered something interesting. Likely, as long as we're using the pkg-config CFLAGS, `#include <blkid.h>` should be good enough. I think the reason this is working on Linux is that `/usr/include` is on the default header search path while the header file may be somewhere other than `/usr/include` on Mac. However, we have the same exact pkg-config file on Linux that you do on Mac. This leads me to believe it's better to just standardize on what pkg-config provides given that it should work on all platforms.

Reply:
Ok, that's great to hear! I will just remove my workaround and adjust the include path then.

I should be able to test the Linux build too.